machine_learning_hep/data/data_prod_20200304/database_ml_parameters_LcpK0spp_20200301.yml

```diff
@@ -212,49 +215,53 @@ LcpK0spp:
 
   multi:
     data:
-      nprocessesparallel: 40
+      nprocessesparallel: 30
       maxfiles: [-1,-1,-1] #list of periods
       chunksizeunp: [100,100,100] #list of periods
       chunksizeskim: [100,100,100] #list of periods
-      fracmerge: [0.1,0.1,0.1] #list of periods
+      fracmerge: [[0.1,0.1,0.1,0.4,1.0,1.0],
+                  [0.1,0.1,0.1,0.4,1.0,1.0],
+                  [0.1,0.1,0.1,0.4,1.0,1.0]]
```

Comment:
This requires the changes in #575 in order to work.

Reply:
Actually, I suspect the changes suggested in #575 might not make it work either because they make `filesel` pt dependent but then `filesel` is used later outside the `for` loop to merge `list_sel_evt` and `list_sel_evtorig` which then should be done also in pt bins I guess.